app/code/Magento/ImportExport/Model/Import/Source/FileParser/UnsupportedPathException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\ImportExport\Model\Import\Source\FileParser;
+
+class UnsupportedPathException extends \RuntimeException
+{
+    private $path;
+
+    public function __construct($path = '', $message = '')
+    {
+        parent::__construct($message ?: sprintf('Path "%s" is not supported', $path));
+
+        $this->path = $path;
+    }
+
+    public function getPath()
```

Comment:
How do you then pass valuable information related to an exception? Shall we then expect message parsing in UI for specific error that needs to be generated from domain exception?